### ISSUE TITLE
[$20]Improper heading level structure.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3245]
**Describe the bug**
Improper heading level structure.

**To Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 
2. Make sure the screen reader is switched on. 
3. Press the letter 'h' to cycle between the headings on the page.

**Expected behavior**

There should be one level 1 heading on the page representing the main content on it. 'Challenges' should be level 1 and 'open challenges' and 'ongoing challenges' should be at heading level 2.  
All sub headings under Open for registration should not be headings

**Actual behavior**
Currently 'open challenges' and 'upcoming challenges' are both being read as level 1 headings.

**Additional context**

Type: Headings

Page: https://www.topcoder.com/challenges

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3245 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3245 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099942 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099942 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099942 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by tosha5252** [comment]
Verified fixed in Chrome using NVDA...screen-reader announces Open for Registration and Ongoing Challenges headings as level 2

Win10

**Comment by lakshmiathreya** [comment]
- 'Open for Registration' and 'Ongoing Challenges' are h2 as expected.
- However, 'Challenge' is not h1 - it is expected as per this requirements of the issue.  Fyi @r0hit-gupta @crazyk07 @sushilshinde @tosha5252

**Comment by r0hit-gupta** [comment]
@lakshmiathreya are you talking about the following 'Challenge' link?

<img width="1275" alt="Screen Shot 2019-09-02 at 4 12 22 PM" src="https://user-images.githubusercontent.com/16841686/64109260-ad718100-cd9c-11e9-813d-a21096cc2fd8.png">

**Comment by tosha5252** [comment]
Heading level for 'Challenges' or 'SRMs' has not been updated.  

Win10/Chrome

**Comment by crazyk07** [comment]
@r0hit-gupta  No need to make "SRMs" a heading, it is just a link.  

But "Challenges" should be the H1 of the page
cc @lakshmiathreya @tosha5252

**Comment by veshu** [comment]
@r0hit-gupta 

Note: Bug bash is closing on 8PM EST, 5 Sept. If this is not fixed by then it will be closed.

**Comment by r0hit-gupta** [comment]
@crazyk07 @veshu fixed. PR  at https://github.com/topcoder-platform/community-app/pull/3296

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099942<br/><br/>```This is an automated message for crazyk via Topcoder X```
